apps-rendering/src/server/server.ts

```diff
@@ -311,6 +312,21 @@ async function serveEditionsArticlePost(
 	}
 }
 
+function getEdition(maybeEdition: string): Edition {
```

Comment:
I'm interested to discover if there's a better way of doing this! It feels verbose (which I like in this case), but maybe it's not the right style.

I also struggled with naming this function & parameter

Reply:
I think this is the correct way of doing this 👍

Possible alternative naming:

```ts
function editionFromString(edition: string): Edition {}
function editionFromString(editionString: string): Edition {}
function editionFromString(str: string): Edition {}
function editionFromString(s: string): Edition {}
```

Reply:
Thanks! I went with `editionFromString(editionString: string)` as well as combining the 'uk' case and `default`